jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/inputs/SliderRange.java

```diff
@@ -11,6 +11,7 @@
 
 public class SliderRange extends UIBaseElement<SliderRangeAssert> {
 
+  private int LocalVariableLeft;
```

Comment:
Please use separate variables in methods instead of adding global variable

Reply:
own local variables used inside methods